src/selectors/chain.ts

```diff
@@ -1,18 +1,18 @@
 import { getContextsSortedAndRanked } from '../selectors'
-import { head, splice } from '../util'
+import { hashThought, head, splice } from '../util'
+import { Path } from '../types'
 import { State } from '../util/initialState'
-import { Path, SimplePath } from '../types'
 
 /** Merges thoughts into a context chain, removing the overlapping head. */
 // use autogenerated rank of context
-// if there is no/empty context chain, return path as-is
-const chain = (state: State, contextChain: SimplePath[], path: SimplePath): Path => {
+// if there is no/empty context chain, return thoughtsRanked as-is
+const chain = (state: State, contextChain: Path[], path: Path) => {
```

Comment:
A `contextChain` should only contain `SimplePaths`, so `Path` is incorrect here. Was there a reason you needed to change the type?

Reply:
Ahh yes! I made a mistake here. Thanks for pointing it out.